testsuite/integration/manualmode/src/test/java/org/jboss/as/test/manualmode/ejb/ssl/SSLRealmSetupTool.java

```diff
@@ -123,75 +116,25 @@ public static ModelNode getRemoteAddress() {
-        result = managementClient.getControllerClient().execute(operation);
-        log.debugf("Setting server-identity ssl for realm %s (password %s, keystore path %s) with result %s", SECURITY_REALM_NAME,
-                SERVER_KEYSTORE_PASSWORD, KEYSTORES_ABSOLUTE_PATH, result.get(OUTCOME));
-        Assert.assertEquals(result.toString(), SUCCESS, result.get(OUTCOME).asString());
-
-        // Adding AUTHENTICATION attribute to SSLRealm
-        operation = new ModelNode();
-        operation.get(OP_ADDR).set(getSecurityRealmsAddressAuthentication());
-        operation.get(OP).set(ADD);
-        operation.get("keystore-path").set(resourcePath + "ejb3/ssl/jbossServer.keystore");
-        operation.get("keystore-password").set(SERVER_KEYSTORE_PASSWORD);
-        operation.get(OPERATION_HEADERS).get(ALLOW_RESOURCE_SERVICE_RESTART).set(true);
-        result = managementClient.getControllerClient().execute(operation);
-        Assert.assertEquals(result.toString(), SUCCESS, result.get(OUTCOME).asString());
+        new SslContextSetup().setup(managementClient.getControllerClient());
 
         // make the https connector use our SSL realm
-        final ModelNode compositeOp = Util.createOperation(COMPOSITE, PathAddress.EMPTY_ADDRESS);
-        final ModelNode steps = compositeOp.get(STEPS);
-
-        operation = new ModelNode();
-        operation.get(OP).set(UNDEFINE_ATTRIBUTE_OPERATION);
-        operation.get(OP_ADDR).add(SUBSYSTEM, "undertow");
-        operation.get(OP_ADDR).add("server", "default-server");
-        operation.get(OP_ADDR).add("https-listener", "https");
-        operation.get(NAME).set("ssl-context");
-        steps.add(operation);
-
-        operation = new ModelNode();
+        ModelNode operation = new ModelNode();
         operation.get(OP).set(WRITE_ATTRIBUTE_OPERATION);
         operation.get(OP_ADDR).add(SUBSYSTEM, "undertow");
         operation.get(OP_ADDR).add("server", "default-server");
         operation.get(OP_ADDR).add("https-listener", "https");
-        operation.get(NAME).set("security-realm");
-        operation.get(VALUE).set(SECURITY_REALM_NAME);
-        steps.add(operation);
-
-        result = managementClient.getControllerClient().execute(compositeOp);
+        operation.get(NAME).set("ssl-context");
+        operation.get(VALUE).set(SSL_CONTEXT_NAME);
+        ModelNode result = managementClient.getControllerClient().execute(operation);
         Assert.assertEquals(result.toString(), SUCCESS, result.get(OUTCOME).asString());
 
         //add remoting connector
         operation = new ModelNode();
         operation.get(OP_ADDR).set(SSLRealmSetupTool.getRemotingConnectorAddress());
         operation.get(OP).set(ADD);
-        operation.get("sasl-authentication-factory").set("application-sasl-authentication");
-        operation.get("ssl-context").set("applicationSSC");
+        operation.get("sasl-authentication-factory").set(SSL_CONTEXT_NAME);
```

Comment:
This can be removed since this test doesn't rely on additional SASL mechanisms being configured.